cmd/river/rivercli/command.go

```diff
@@ -124,7 +125,7 @@ func RunCommand[TOpts CommandOpts](ctx context.Context, bundle *RunCommandBundle
 
 	ok, err := procureAndRun()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed: %s\n", err)
+		fmt.Fprintf(os.Stdout, "failed: %s\n", err)
```

Comment:
I'm surprised by this change.

If you had used command.StdOut, I might have understand, but here reporting errors to stdout seems strange.

Maybe I simply missed something about your PR

If it's OK to use stdout here for error reporting, adding an inline comment could help

Reply:
Good catch. That's actually an accident — I was refactoring lots of stuff related to output to find a good way of capturing it for comparison in tests, and this accidentally got hit. Opened #547 with a fix.

Reply:
Thanks